diagnostic.md

```diff
@@ -4,7 +4,10 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+Class=' emoji-size-boost is-u2f-enabled'
+id='js-pjax-loader-bar'
+div class='repo-file-upload-tree-target js-document-dropzone js-upload-manifest-tree-view'
+id=readme
```

Comment:
We want selectors such as `body`, `.repohead`, or `div#files_bucket`, as examples.